theme/templates/page.html

```diff
@@ -3,5 +3,15 @@
 {% block title %}{{ page.title }}{% endblock title %}
 
 {% block content %}
-    {{ page.content }}
+	{% if page.body_class %}
+		<!-- do nothing -->
+	{% else %}
+		<section class="community">
```

Comment:
This class is different from what is on the original HTML page. You have to make the two be exactly the same if you want to keep the existing look and feel of the page.